sensorprocess/testhelper.go

```diff
@@ -0,0 +1,269 @@
+package sensorprocess
```

Comment:
The contents in this file were extracted from `sensorprocess_test.go` to make the file smaller and more readable. No changes were made to the contents in this file.